op-monitorism/transaction_monitor/cli.go

```diff
@@ -0,0 +1,91 @@
+package transaction_monitor
+
+import (
+	"fmt"
+	"os"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	L1NodeURLFlagName  = "l1.node.url"
```

Comment:
Can use this on L2 so can remove L1 from the name

Reply:
will remove